Rearrange & edit "Creating a Segment" sections (#6829)
braze-inc/braze-docs: _docs/_hidden/private_betas/updated_segment_experience.md

Segments already using the Segment Membership Filter cannot be further included or nested into other segments.
{% endalert %}

#### Exclusion groups (optional)

When building a segment, you can apply one or multiple exclusion groups. Exclusion groups contain criteria that identify users to exclude from your segment, and will always be connected to your filter groups with an "AND NOT" operator.

If a user falls into your exclusion group criteria, they will not be part of your segment, even if they meet the criteria within your filter groups.

> Create an exclusion group by adding filters, the same way you would for filter groups. The estimated excluded users statistic is the number of users that would normally be targeted by this segment, but are excluded because of exclusion group criteria.

Reviewer: ```suggestion
Create an exclusion group by adding filters like you would for filter groups. The estimated excluded users statistic is the number of users normally targeted by this segment but are excluded because of exclusion group criteria.
```

> Excluded users will not be counted as part of your segment’s **Total reachable users** statistic.

Reviewer: ```suggestion
Excluded users will not be counted as part of your segment’s _Total reachable users_ statistic.
```
Is this a metric? 🤔 if not, disregard the style suggestion
